cg/meta/workflow/nf_analysis.py

```diff
@@ -803,3 +835,68 @@ def get_cases_to_store(self) -> list[Case]:
             if self.trailblazer_api.is_latest_analysis_completed(case_id=case.internal_id)
             or self.trailblazer_api.is_latest_analysis_qc(case_id=case.internal_id)
         ]
+
+    def get_reference_genome(self, case_id: str) -> GenomeVersion:
```

Comment:
Adding this method will create two parallel implementations, as we have `get_genome_build` in the report API (this one should've been added to the analysis API 😓)

Reply:
Also this part I'll try to fix in my PR; move all these analysis level methods from the delivery report to the analysis API @fevac

Reply:
merged and converged with your PR